development/tools/jobs/tester/tester.go

```diff
@@ -49,6 +49,8 @@ const (
 	ImageBootstrap20181204 = "eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181204-a6e79be"
 	// ImageBootstrap20190604 represents boostrap image published on 2019.06.04
 	ImageBootstrap20190604 = "eu.gcr.io/kyma-project/test-infra/bootstrap:v20190604-d08e7fe"
+	// ImageBootstrap20201009 represents boostrap image published on 2020.10.09
+	ImageBootstrap20201009 = "eu.gcr.io/kyma-project/test-infra/bootstrap:v20201009-ec3cc352"
```

Comment:
Why was there a need to introduce another bootstrap image?

Reply:
I do not introduce a new bootstrap image just added a constant for the newest one so I can assert it is used in the job as part of the tests.

Reply:
I understand, however it will be needed to make changes in jobs to use only the latest bootstrap image available. But it's the case we can handle separately.